stock_product_set/views/stock_picking_views.xml

```diff
@@ -11,7 +11,7 @@
                     type="action"
                     name="%(act_open_wizard_product_set_add_from_picking)d"
                     icon="fa-cubes"
-                    attrs="{'invisible': ['|', ('state', '!=', 'draft'), ('immediate_transfer', '=', True)]}"
```

Comment:
The field immediate_transfer has disappeared?

Reply:
Yes, it is actually removed already in 17.0.: https://github.com/OCA/OpenUpgrade/blob/857b65680f0033e2df07497d74e69c2ebb25d28a/openupgrade_scripts/scripts/stock/17.0.1.1/upgrade_analysis_work.txt#L64